comment/views.py

```
@@ -0,0 +1,190 @@
+        manual_parameters=[post_id_param], 
+        responses={
+            200: CommentSerializer(many=True),
+            404: "Not Found",
+        },
+    )
+    def get(self, request):
+        postId = request.GET.get("post")
+
+        try:
+            comments = Comment.objects.filter(post = postId)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)